linux/LinuxProcessList.c

```diff
@@ -1312,6 +1312,9 @@ static bool LinuxProcessList_recurseProcTree(LinuxProcessList* this, openat_arg_
    const ScreenSettings* ss = settings->ss;
    const struct dirent* entry;
 
+   /* set runningTasks from /proc/stat (from Machine_scanCPUTime) */
+   pl->runningTasks = lhost->running;
+
```

Comment:
With `ProcessList_goThroughEntries` being the entry point calling `LinuxProcessList_recurseProcTree` it would be better to place this initialization there, thus executing it only once (`PL_gTE` recurses, thus executing this code potentially multiple times; doesn't hurt, but isn't necessary either).

Reply:
Hmm, maybe - I did prefer this location because it keeps the two matched comments about runningTasks in that same routine (the second is towards the end of the loop, where the runningTasks++ used to be, back in the day).

I don't think its a problem to execute an integer assignment multiple times, especially given everything else going on in that routine - better to keep the code intent clear and understandable IMO.